README.md

```diff
@@ -13,7 +13,7 @@
 
 <p align="center"><b>PACSTALL</b></p>
 <p align="center">
-<a href="https://github.com/pacstall/pacstall"><img align="center" src="https://pacstall.dev/image/pacstall.svg" alt="Pacstall Logo"></a>
+<a href="https://github.com/pacstall/pacstall"><img align="center" src="https://pacstall.dev/pacstall.a6e2442a.svg" alt="Pacstall Logo"></a>
```

Comment:
Use the logo link from the `python3-rewrite` branch, as it uses the github link which is at the moment faster.